content/2017/1/25/parallax.rst

```diff
@@ -0,0 +1,36 @@
+public: yes
+author: sondra
+tags: [design]
+image:
+  - src: '2017/parallax/parallax.jpg'
+summary: |
+  Whether you like it not, parallax scrolling is a reality, so let's learn how to do it well with CSS 3D. Paul Lewis' guide to performant parallaxing is the perfect place to start.
```

Comment:
I'm not a fan of the judgement/battle implied by the 'like it or not' start. I'd leave that out and focus on what there is to learn.